rules-tests/Php80/Rector/ClassMethod/AddParamBasedOnParentClassMethodRector/Fixture/fixture_has_parent_typed_param.php.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Rector\Tests\Php80\Rector\ClassMethod\AddParamBasedOnParentClassMethodRector\Fixture;
+
+use Rector\Tests\Php80\Rector\ClassMethod\AddParamBasedOnParentClassMethodRector\Source\ParentWithTypedParam;
+
+// ref https://3v4l.org/4Mgrt
+class B extends ParentWithTypedParam{
+    public function execute()
+    {
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\Php80\Rector\ClassMethod\AddParamBasedOnParentClassMethodRector\Fixture;
+
+use Rector\Tests\Php80\Rector\ClassMethod\AddParamBasedOnParentClassMethodRector\Source\ParentWithTypedParam;
+
+// ref https://3v4l.org/4Mgrt
+class B extends ParentWithTypedParam{
+    public function execute($foo)
```

Comment:
Here should be `int $foo`, to respect parent type. We should keep code as close as possible to human change.